sunpy/map/sources/tests/test_suvi_source.py

```diff
@@ -0,0 +1,44 @@
+"""
+Test cases for SUVI Map subclass.
+"""
+
+import os
+import glob
+
+import pytest
+
+from sunpy.map.sources.suvi import SUVIMap
+from sunpy.map import Map
+import sunpy.data.test
+
+path = sunpy.data.test.rootdir
+fitspath = glob.glob(os.path.join(path, "test_suvi_image.fits"))
```

Comment:
Is this file included in this PR? It doesn't look like it.

Reply:
It is not